src/dex-ui/routes/router.tsx

```diff
@@ -38,17 +41,17 @@ export const router = createBrowserRouter(
       <Route path={`${Paths.Pools.AddLiquidity}/:pairId/:poolName`} element={<AddLiquidityPage />} />
       <Route path={Paths.Pools.Withdraw} element={<WithdrawPage />} />
       <Route path={Paths.Pools.CreatePool} element={<CreatePoolPage />} />
-      {/* TODO: Create governance route heirarchy. */}
-      <Route path={Paths.Governance.default} element={<Governance />} />
-      <Route path={`${Paths.Governance.ProposalDetails}/:id`} element={<ProposalDetails />} />
-      <Route path={Paths.Governance.SelectProposalType} element={<SelectProposalType />} />
-      <Route path={Paths.Governance.CreateNewToken} element={<CreateProposal proposalType="new-token" />} />
-      <Route path={Paths.Governance.CreateText} element={<CreateProposal proposalType="text" />} />
-      <Route path={Paths.Governance.CreateTokenTransfer} element={<CreateProposal proposalType="token-transfer" />} />
-      <Route
-        path={Paths.Governance.CreateContractUpgrade}
-        element={<CreateProposal proposalType="contract-upgrade" />}
-      />
+      <Route path={Paths.Governance.default}>
+        <Route index element={<Governance />} />
+        <Route path={`${Paths.Governance.ProposalDetails}/:id`} element={<ProposalDetails />} />
+        <Route path={Paths.Governance.SelectProposalType} element={<Outlet />}>
```

Comment:
My Bad, I meant to remove it since we do not have any shared layout(earlier I was planning to make a shared layout for 4 proposal forms which would share the common header layout but it was increasing the complexity of the routes tree) among them its of no use here, thanks for highlighting this one.